### PR TITLE
KOGITO-3589 Extract apps from kogito-runtimes

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3589

Dependent PRs:
- https://github.com/kiegroup/kogito-pipelines/pull/89
- https://github.com/kiegroup/kogito-runtimes/pull/909
- https://github.com/kiegroup/kogito-apps/pull/554

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: Jenkinsfile.deploy [hunk 1]
```diff
@@ -69,29 +71,20 @@ pipeline {
...
             }
             steps {
                 prepareForPR('kogito-runtimes')
-                prepareForPR('kogito-apps')
             }
         }
-        stage('Setup Maven release config'){
-            when {
-                expression { return isRelease() }
-            }
```
Comment (Kevin-Mok) [inline Jenkinsfile.deploy:90]: How come this is removed now?

Comment (radtriste) [reply Jenkinsfile.deploy:90]: The Maven config will be set for all jobs running, not only for the release

**Comment by radtriste** [comment]
Jenkins retest this

**Comment by radtriste** [comment]
testing in progress

**Review by Kevin-Mok — APPROVED** [review]

**Review by rsynek — APPROVED** [review]

**Comment by radtriste** [comment]
Testing ok https://docs.google.com/spreadsheets/d/1L4NvDuQ6h4nOVuXMgpVmlvF-sjoTH6eI6EJFLWiwe7I/edit?usp=sharing
@rsynek good to merge

**Pull request merged**